packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -653,6 +653,15 @@ blockquote {
   padding: 0;
 }
 
+.mainContainer .wrapper .post .postHeader:before, .mainContainer .wrapper .post .postHeaderTitle:before {
```

Comment:
I was actually following the same hierarchy that was already in place for styling the post title  so that it's all consistent. If you still want me to use "less-specific" selector, I can go ahead and push another update.

Reply:
> Separate selectors should be on its own line.

@yangshun - Just for clarification, did you mean writing it like this?

```css
.mainContainer .wrapper .post .postHeader:before,
.mainContainer .wrapper .post .postHeaderTitle:before {
....
}
```

> We shouldn't be writing the CSS like this. With such a specific hierarchy, this is bound to break if we change the DOM structure. Can we use a less-specific selector?

As I mentioned in my previous reply, I'm following the CSS structure that was there initially. Do you still want me to update my changes to use less-specific selector?

Reply:
> Just for clarification, did you mean writing it like this?

I'm not yangshun, but yes, that’s exactly what was meant.

> As I mentioned in my previous reply, I'm following the CSS structure that was there initially. Do you still want me to update my changes to use less-specific selector?

Fair enough, better following current CSS structure.

Reply:
Thanks for the confirmation @lex111 . I'll update the css so that different selectors are on separate line.